test/cctest/node_test_fixture.h

```diff
@@ -0,0 +1,85 @@
+#ifndef TEST_CCTEST_NODE_TEST_FIXTURE_H_
+#define TEST_CCTEST_NODE_TEST_FIXTURE_H_
+
+#include <stdlib.h>
+#include "gtest/gtest.h"
+#include "node.h"
+#include "env.h"
+#include "v8.h"
+#include "libplatform/libplatform.h"
+
+using node::Environment;
+using node::IsolateData;
+using node::CreateIsolateData;
+using node::CreateEnvironment;
+using node::AtExit;
+using node::RunAtExit;
+
+class ArrayBufferAllocator : public v8::ArrayBuffer::Allocator {
+ public:
+  virtual void* Allocate(size_t length) {
+    return AllocateUninitialized(length);
+  }
+
+  virtual void* AllocateUninitialized(size_t length) {
+    return calloc(length, sizeof(int));
```

Comment:
Actually not a typeo but that does not mean it is correct :) 
When reading the documentation for calloc the size argument takes a size_t and I though sizeof would be appropriate. Are there better/correct ways of doing this?